lib/widgets/action_sidebar.dart

```diff
@@ -0,0 +1,219 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class ActionSideBar extends StatefulWidget {
+  const ActionSideBar(this.actionButtons);
+
+  final List<Widget> actionButtons;
+
+  @override
+  _ActionSideBarState createState() => _ActionSideBarState();
+}
+
+class _ActionSideBarState extends State<ActionSideBar>
+    with SingleTickerProviderStateMixin {
+  _ActionSideBarState({this.isExtended = false});
```

Comment:
What's the point of the parameter in this constructor? It isn't actually used anywhere.

If you just want to define the default value, just do `bool isExtended = false` below.

Reply:
Yes, it's not really used so could be moved to field:

https://github.com/student-hub/acs-upb-mobile/pull/250/commits/8c0e90bd1db807c70ea5422527345d766711da4c#diff-91af2d9745e86244940959ec39c99cca9b8b43c1502042d85f16fcc6a311030bR17